ghost/admin/lib/asset-delivery/index.js

```diff
@@ -43,6 +43,8 @@ module.exports = {
                 for (const [key, value] of Object.entries(this.packageConfig)) {
                     console.log(`Asset-Delivery: ${key} = ${value}`);
                 }
+
+                this.packageConfig[`adminXActivitypubCustomUrl`] = 'https://cdn.jsdelivr.net/npm/@tryghost/admin-x-activitypub@0.x/dist/admin-x-activitypub.js'
```

Comment:
Does this work? 🤔 I thought it had to be `...@~0.0/..`

Reply:
It is working - the docs say it should be `@0` or `@0.0` though - I'm gonna change it to that